### DISCUSSION TITLE
Transformation/Upright "update" after pasting settings

### DISCUSSION THREAD

**Discussion by flyingpaul1980** [rsjaffe/MIDI2LR#622] [Q&A]
Hi, it would be super useful to be able to bind the command to update upright settings in Midi2LR, but i cannot find the respective command in the list..

could you please enlighten me, which would be the correct binding?

Thanks in advance :)

**Comment by antzpantz** [comment]
It's all under Transform:

https://github.com/rsjaffe/MIDI2LR/wiki/Commands

**Reply by flyingpaul1980** [reply]
well feel free to show me, because i don't see the desired function in that list :(

**Reply by rsjaffe** [reply]
See https://github.com/rsjaffe/MIDI2LR/wiki/Commands#17

**Reply by flyingpaul1980** [reply]
> See https://github.com/rsjaffe/MIDI2LR/wiki/Commands#17

![2022-02-01 11_09_41-Catalog-2-2-v10 - Adobe Photoshop Lightroom Classic - Develop](https://user-images.githubusercontent.com/76615610/151949642-2c99b361-748b-465c-af16-a03e2ef45f3b.jpg)

i was referring to this button, but i guess the same as auto straighten, adobe does not provide a function here

**Comment by flyingpaul1980** [comment]
also the "auto straighten" button in the crop window would be nice :)

**Reply by rsjaffe** [reply]
That's unavailable in MIDI2LR as Adobe doesn't provide a way to use that function.
